resources/qml/Settings/SettingTextField.qml

```diff
@@ -92,6 +92,8 @@ SettingItem
 
         UM.Label
         {
+            id: unitlabel
```

Comment:
```suggestion
            id: unitLabel
```

Reply:
That's OK! -- I can't directly commit this, since I end up using it in the binding below and that needs to be changed too then.

Hm, maybe I can try suggesting the other needed change myself and then bundling those changes 🤔

Reply:
Yeah I basically used the suggestion feature to make it clear, but you don't HAVE to use it :slightly_smiling_face:

Reply:
I figured, but it _is_ very easy when you can use them 😁.